systemtests/custom-operator-registry/build_push_registry.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+FROM=${1}
+CUSTOM_IMAGE=${2}
+
+CONTAINER_CTL=${CONTAINER_CTL:-docker}
```

Comment:
Please use DOCKER env var like we use in all makefiles :) see Makefile.env.mk

Reply:
done :)